src/main/java/seedu/condonery/MainApp.java

```diff
@@ -58,7 +62,9 @@ public void init() throws Exception {
         UserPrefs userPrefs = initPrefs(userPrefsStorage);
         PropertyDirectoryStorage propertyDirectoryStorage =
             new JsonPropertyDirectoryStorage(userPrefs.getPropertyDirectoryFilePath());
-        storage = new StorageManager(propertyDirectoryStorage, userPrefsStorage);
+        ClientDirectoryStorage clientDirectoryStorage =
+                new JsonClientDirectoryStorage(userPrefs.getClientDirectoryFilePath());
```

Comment:
The indentation level is inconsistent here (8 spaces here, but 4 spaces on line 64), but im not sure which is the correct level. I think its a wrapped line, so it should be 8?